api/scpca_portal/readme_file.py

```diff
@@ -38,3 +38,20 @@ def get_file_contents(download_config: Dict, project) -> str:
             "project_url": project.url,
         },
     ).strip()
+
+
+# NOTE: Temporarily added this until all consolidate readme PRs are finalized and merged
```

Comment:
As per our 1:1 talk, I updated `NOTE` to `TODO` in the comments and instead of creating a new issue, I linked the PR that needed to be merged first. Thank you, David!